asset_dashboard/static/js/PortfolioPlanner.js

```diff
@@ -81,7 +88,7 @@ class PortfolioPlanner extends React.Component {
           senate_districts: project.senate_districts,
           commissioner_districts: project.commissioner_districts,
           key: funding.id,
-          funding_source: funding['source_type'],
+          funding_source: fundingSourceLabel,
```

Comment:
Making sure to show the readable label instead of the snake_cased one on the table